components/channel_header/channel_header.tsx

```diff
@@ -623,10 +723,8 @@ class ChannelHeader extends React.PureComponent {
 
             ariaLabel = formatMessage(formattedMessage).toLowerCase();
             toggleFavoriteTooltip = (
-                <Tooltip id='favoriteTooltip' >
-                    <FormattedMessage
-                        {...formattedMessage}
-                    />
+                <Tooltip id='favoriteTooltip'>
+                    <FormattedMessage {...formattedMessage}/>
```

Comment:
not directly related, but the variable name is misleading: it should be `formattedMessageDescriptor`